tests/testthat/testImportSNV.TCGA.R

```diff
@@ -0,0 +1,208 @@
+    trueOutPre <- read.csv("testOutPre", skip=7, sep = "\t")
+
+    # run the module on fMAF and rSNV as parameters
+    moduleOutPre <- importSNV(c("testInPre"), "module02")
+    modulePre <- read.csv("moduleOutPre", skip=7, sep = "\t")
+    # add to prexisting rSNV file
+    all.equal(trueOutPre, modulePre)
+
+    # run the module on fMAF and rSNV and silent as parameters
+    moduleOutPre <- importSNV(c("testInPre"), "module02", silent=TRUE)
+    modulePre <- read.csv("moduleOutPre", skip=7, sep = "\t")
+    # add to prexisting rSNV file
+    all.equal(trueOutPre, modulePre)
+
+    # run the module on fMAF, rSNV, silent and writeLog as parameters
+    moduleOutPre <- importSNV(c("testInPre"), "module02", silent=TRUE, writeLog=TRUE)
+    modulePre <- read.csv("moduleOutPre", skip=7, sep = "\t")
+    # add to prexisting rSNV file
+    all.equal(trueOutPre, modulePre)
+
+    # test input with rSNV as input and NA values
+    testInNA <- "~/Desktop/rete/testInNA"
+    # test output for rSNV as input and NA values
+    sample1 <- c("VWA5B1", "chr1",	20319393,	20319393,	"+",	"Missense_Mutation",	"SNP", 	"C",	"C",	"T", "TCGA-QQ-A8VF-01A-11D-A37C-09", "2109690d-e8b1-4b5c-9a39-21af273324cf")
+    write(sample1, "testOutNA", ncolumns=12, sep="\t", append=TRUE)
+    trueOutNA <- read.csv("testOutNA", skip=7, sep = "\t")
+
+    # run the module on fMAF, rSNV, silent, writeLog and na.rm as parameters
+    moduleOutNA <- importSNV(c("testInNA"), "moduleNA", silent=TRUE, writeLog=TRUE, na.rm=TRUE)
+    moduleNA <- read.csv("moduleOutNA", skip=7, sep = "\t")
+    # add to prexisting rSNV file
+    all.equal(trueOutNA, moduleNA)
+})
+
+
+test_that("a corrupt input does not lead to corrupted output", {
+    # if fMAF file open does not follow the 2.4 or agreed upon version either STOP or DROP file
+    expect_error(importSNV(c("testIn2.4.1")))
+    # if fMAF file open does not have the required columns at the
+    #           right column number STOP or DROP file
+    expect_error(importSNV(c("testInWrongColumns")))
+    # if rSNV is provided, check if number of columns in rSNV is worng
+    expect_error(importSNV(c("testIn")), "testOutINVALIDcolumns")
+    # if rSNV is provided, check when header labels are wrong
+    expect_error(importSNV(c("testIn")), "testOutINVALIDheaders")
+})
+
+test_that("silent and write works as intended", {
+    # if silent=TRUE, check if output to console is supressed
+    testF <- tempfile()
+    capture.output(Do.A.Thing(silent = TRUE), file = testF)
+    expect_equal(length(readLines(testF)), 0)
+    expect_true(file.remove(testF))
+
+    # if writeLog=TRUE, check if log file is created
+
+    # if writeLog=TRUE, check if logs are written to it correctly
+
+})
+
+
+test_that("test that columns contains valid values", {
+    # check if variant_class contains valid values from known list
+    ### will finalize these tests once import module is implemented ###
+
+    classColumn <- addColumn("testOut")
```

Comment:
I'm not completely sure what you are doing here - but this looks like a test that actually belongs in your function. Rather than iterate through the choices, provide one test input that contains each of the options, and test once that your functions passes all of them. Then provide *one* wrong keyword, and confirm that the import fails.

Also, use ```%in%``` to check whether some observed keywords are in a list, not iteration. Something like:
```
myVclasses <- unique(data$variant.class)
if (! all(myVclasses %in% knownVclasses)) { ... that's an error.
```